src/app/component/places/places.component.html

```diff
@@ -0,0 +1,78 @@
+          <datalist id="places_names">
+            <option *ngFor="let card of cardsCollection" value="{{card.cardName}}">{{card.cardText}}</option>
+          </datalist>
+          <select class="search-chooser">
+            <option value="Lviv">Lviv</option>
+          </select>
+        </div>
+      </div>
+      <div class="filter">
+        <app-tag-filter (tagsList)="getFilterData($event)" [header]="'homepage.eco-news.filter'"
+          [storageKey]="'Places-filter'" [tagsListData]="tagList"></app-tag-filter>
+      </div>
+    </div>
+    <hr>
+    <div class="places-map">
+      <div class="place-info">
+        <mat-drawer-container class="example-container sidebar-container" autosize>
+          <mat-drawer #drawer class="example-sidenav sidebar-drawer" mode="side" closed>
+            <div class="sidebar-content">
+              <span class="close-mark" (click)="drawer.toggle()"></span>
+              <h6 class="content-title">{{contentObj.cardName}}</h6>
+              <img src="{{contentObj.cardImgUrl}}" alt="content-image" class="card-img-top">
+              <span class="content-address">{{contentObj.cardAddress}}</span>
+              <!-- <img class="card-bookmark" src="assets/img/icon/bookmark_default.svg" alt="bookmark"> -->
+              <p class="content-description">{{contentObj.cardText}}</p>
+              <span class="content-website">https://placewebsite.com</span>
+              <ul class="content-list">
+                <li class="content-list-event">
+                  <img class="event-icon" src="{{notification}}" alt="ev-icon">
+                  <p class="event-description">Special offer - discount for bonus card holders</p>
+                </li>
+                <li class="content-list-event">
+                  <img class="event-icon" src="{{share}}" alt="ev-icon">
+                  <p class="event-description">Place profile on Greencity</p>
+                </li>
+              </ul>
+              <h6 class="content-reviews-title">Rating &nbsp;&nbsp; <span>{{contentObj.cardRating}}</span></h6>
+              <div class="stars-container">
+                <img *ngFor="let star of contentObj.cardStars" class="star" src="{{star}}" alt="star">
+              </div>
+              <span class="content-reviews-amount">20 reviews</span>
+            </div>
+          </mat-drawer>
+        </mat-drawer-container>
+      </div>
+      <div class="google-map" #map>
+        <agm-map [style.width.%]="100" [style.height.%]="100" [latitude]="lat" [longitude]="lng" [zoom]="zoom">
+          <agm-marker *ngFor="let marker of markerListCopy" [latitude]="marker.lat" [longitude]="marker.lng"
+            (markerClick)="getMarker($event)" [iconUrl]="marker.iconUrl" class="marker"></agm-marker>
+        </agm-map>
+      </div>
+    </div>
+    <div class="places-recommendation">
+      <h6 class="recommendation-title">Popular eco places</h6>
```

Comment:
move text to assest folder
so user will be able to see translated text to different languages

Reply:
DONE ✔️